content/en/docs/setup/best-practices/certificates.md

```diff
@@ -19,16 +19,23 @@ This page explains the certificates that your cluster requires.
 
 ## How certificates are used by your cluster
 
-Kubernetes requires PKI for the following operations:
-
-* Client certificates for the kubelet to authenticate to the API server
-* Server certificate for the API server endpoint
-* Client certificates for administrators of the cluster to authenticate to the API server
-* Client certificates for the API server to talk to the kubelets
-* Client certificate for the API server to talk to etcd
-* Client certificate/kubeconfig for the controller manager to talk to the API server
-* Client certificate/kubeconfig for the scheduler to talk to the API server.
-* Client and server certificates for the [front-proxy](/docs/tasks/extend-kubernetes/configure-aggregation-layer/)
+Kubernetes requires certificates to allow the following authentication to take place:
+
```

Comment:
Hi @RobertKielty . Good ideas.
Here is another way to reorganize the content.

```
In Kubernetes, cluster components require different PKI certificates for authentication.

1. Client certificates authenticate between:
    * the kubelet and the API server
    * cluster administrators and the API server
    * the API server to the kubelets
    * the API server to etcd
1. Server certificates authenticate between:
    * the API server endpoint and connecting clients
1. Client certificate/kubeconfig allows:
    * the controller manager to talk to the API server
    * the scheduler to talk to the API server.
1.  Client and server certificates for the [front-proxy](/docs/tasks/extend-kubernetes/configure-aggregation-layer/)
```

Reply:
Thank you for this feedback.
 
I like the use of the use of the full-colon instead of the ellipsis.

I do not like the use of "authentication between" instead of "authenticates subject component to the object component" as I it muddies the waters as to what entity is being authenticated and what entity is seeking the authentication.

Reply:
it would be good to include a reference to the [Kubernetes Component Diagram](https://kubernetes.io/docs/concepts/overview/components/) as really what are doing here is traversing that diagram in terms of PKI requirements.

Reply:
That's fine @RobertKielty . Use the wording that you feel best.